collections/_news/2020-06-01-meshery-accepted-into-cncf-landscape.md

```diff
@@ -2,10 +2,27 @@
 layout: post
 type: news
 date:   2020-06-01 12:15:05 +0000
-title:  "Meshery accepted into CNCF Landscape"
+title:  "Meshery accepted into the CNCF Landscape"
 author: the Newsroom
 image: https://raw.githubusercontent.com/cncf/artwork/master/other/cncf-landscape/stacked/color/cncf-landscape-stacked-color.svg
 #permalink:  https://landscape.cncf.io/category=service-mesh&format=card-mode&grouping=category&selected=meshery
 eurl:  https://landscape.cncf.io/category=service-mesh&format=card-mode&grouping=category&selected=meshery
 ---
-<img src="/assets/images/posts/2019-11-15-layer5-joins-the-cncf/Meshery accepted into the CNCF Landscape.png" width="8vw" />
\ No newline at end of file
+<div class="center" style="color:#00b39f;position:relative;top:-10px;font-size:1.00em;">A story about growing together as a community and building a project that matters</div>
+
+Meshery took its first baby steps into the Cloud native world in the February of 2019, aiming to provision and manage the lifecycle of service meshes and establish its prowess as the service management plane and now, barely a year later, we are bursting with pride as we make this announcement.
```

Comment:
- First and baby are redundant.
- Capitalize both cloud and native or neither.
- It’s not “the “February, it’s just February.
- Good job on outlining what the project is aiming to do. that’s not an easy thing to capture. I’ll provide a rewrite here.
- instead of “as we make this announcement “, “as we announce"
Capitalize landscape.
Meshery is not a service mesh. It was excepted into the service mesh category of the landscape. 
Remove "most" in "most valued"
Go ahead and include Anton’s for post here. We don’t need to link to the original article. Moreover, if you’re going to post his article as a blog here, you would just want to link to that.